task2/src/main/java/Accommodation.java

```diff
@@ -0,0 +1,72 @@
+import electricalAppliances.ElectricalAppliances;
+
+import java.util.ArrayList;
+
+public class Accommodation {
+    protected ArrayList<ElectricalAppliances> accommodationDevices = new ArrayList<ElectricalAppliances>();
+
+    public Accommodation(ArrayList<ElectricalAppliances> devices) {
+        this.accommodationDevices = devices;
+    }
+
+
+    public static ArrayList<ElectricalAppliances> getPowerOnDevices(ArrayList<ElectricalAppliances> accommodationDevices) {
```

Comment:
You shouldn't use any parameters for this method. You already have ElectricalAppliances list in this class.